<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>batchDownloadForm</class>
 <widget class="QWidget" name="batchDownloadForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>150</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>330</width>
    <height>140</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>330</width>
    <height>150</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>批量下载</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>15</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>选择路径</string>
   </property>
  </widget>
  <widget class="QToolButton" name="btnChooseDir">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>15</y>
     <width>37</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblDir">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>17</y>
     <width>191</width>
     <height>55</height>
    </rect>
   </property>
   <property name="text">
    <string>无</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>83</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>注：在点击下载前，请务必确保自己配置的IDM是有效的！
桌面上一定要有已抓取的download.txt文件</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOK">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>120</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCheckIDM">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>检查IDM</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCheckLinks">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>120</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>检查链接</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
